clang/File.d

```diff
@@ -13,8 +13,13 @@ struct File
 {
     mixin CX;
 
-    @property string name ()
+    string name() const
     {
-        return toD(clang_getFileName(cx));
+        return toD(clang_getFileName(cast(void*) cx));
```

Comment:
I'm not keen on casting away const, is it really necessary to make `name` const?